<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Memorija</class>
 <widget class="QWidget" name="Memorija">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>751</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item row="1" column="1">
     <widget class="QWidget" name="widget_7" native="true"/>
    </item>
    <item row="2" column="1">
     <widget class="QWidget" name="widget_12" native="true"/>
    </item>
    <item row="1" column="2">
     <widget class="QWidget" name="widget_8" native="true"/>
    </item>
    <item row="2" column="4">
     <widget class="QWidget" name="widget_15" native="true"/>
    </item>
    <item row="3" column="0">
     <widget class="QWidget" name="widget_16" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_2" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_6" native="true"/>
    </item>
    <item row="1" column="4">
     <widget class="QWidget" name="widget_10" native="true"/>
    </item>
    <item row="3" column="2">
     <widget class="QWidget" name="widget_18" native="true"/>
    </item>
    <item row="3" column="3">
     <widget class="QWidget" name="widget_19" native="true"/>
    </item>
    <item row="0" column="4">
     <widget class="QWidget" name="widget_5" native="true"/>
    </item>
    <item row="1" column="3">
     <widget class="QWidget" name="widget_9" native="true"/>
    </item>
    <item row="3" column="1">
     <widget class="QWidget" name="widget_17" native="true"/>
    </item>
    <item row="2" column="3">
     <widget class="QWidget" name="widget_14" native="true"/>
    </item>
    <item row="3" column="4">
     <widget class="QWidget" name="widget_20" native="true"/>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="widget_3" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="widget_11" native="true"/>
    </item>
    <item row="2" column="2">
     <widget class="QWidget" name="widget_13" native="true"/>
    </item>
    <item row="0" column="3">
     <widget class="QWidget" name="widget_4" native="true"/>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget_1" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="userName1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 255, 127)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="userName2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 255, 127)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user2points">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 255, 127)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user1points">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 255, 127)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
